### PR TITLE
 Documentation on bitwise operator

### PR DESCRIPTION
**Pull request by Akshat-2002-coder**
<hr>

documentation on bitwise operators and some useful results based on it.

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes # <3359> 

<hr>

## Type of change 📝

- [] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

**Comment by ipsitagoel** [comment]
@Akshat-2002-coder This topic has already been covered previously and added to the repository: https://github.com/girlscript/winter-of-contributing/blob/C_CPP/C_CPP/Fundamentals/Operators%20%26%20Expressions/operators-%26-expressions-in-C.md

- You can either explore further on Bitwise operators such as Bitmasking etc.
- Or you can contribute to some other topic.

**Comment by Akshat-2002-coder** [comment]
@ipsitagoel  mam can u plz tell from where I can choose topics which are not taken up by someone else

**Comment by ipsitagoel** [comment]
@Akshat-2002-coder You can refer to the C/CPP roadmap and check if content related to that particular topic has already been added to the repo or if an issue/PR has been raised.
Link to the roadmap: https://github.com/girlscript/winter-of-contributing/tree/C_CPP/C_CPP

File: C_CPP/Bitwise operators/bitwise.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+   int x=4;
+   cout<<(x>>1); // output is 2
+```
+- **" << " left shift operator** :- It is a unary operator which is used to shift the bit position towards left. for example:-<br>
+4<<1 :- means in binary representation 4 each bit will be shifted towards left by 1 unit.
+```
+   int x=4;
+   cout<<(x<<1); // output is 8
+```
+- **~ (bitwise NOT)** :- It is also a unary operator which is used to complement the bits, means if it is '0' it becomes '1' and vice versa.
```
Comment (kashishahuja2002) [inline C_CPP/Bitwise operators/bitwise.md:40]: It hasn't been bold.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Try adding some code snippets.

**Comment by Akshat-2002-coder** [comment]
@Saviour1001  sir can u plz review my pull request now

**Comment by Saviour1001** [comment]
As suggested by the discussion above the explanation of bitwise operator exists [here](https://github.com/girlscript/winter-of-contributing/blob/C_CPP/C_CPP/Fundamentals/Operators%20%26%20Expressions/operators-%26-expressions-in-C.md). I liked the extra points you added in the end. So add a file named "Bitwise Operator in detail" in this folder and you can add your content which is already not present. 
Add your content in this [folder](https://github.com/girlscript/winter-of-contributing/tree/C_CPP/C_CPP/Fundamentals/Operators%20%26%20Expressions)

**Comment by Akshat-2002-coder** [comment]
@Saviour1001  sir can u plz review it now😅.

**Comment by Saviour1001** [comment]
image
You added an empty file, remove the previous file as well. Double check your content before pinging.

**Comment by Akshat-2002-coder** [comment]
@Saviour1001  sorry for inconvenience now I have double checked myself.

**Pull request merged**